assets/queries/terraform/aws/rds_with_backup_disabled/query.rego

```diff
@@ -3,45 +3,6 @@ package Cx
 import data.generic.common as common_lib
 import data.generic.terraform as tf_lib
 
```

Comment:
Hello again, @patrickpichler! Thank you for another contribution!

In addition to Miguel's suggestion, do you mind also updating the Ansible query, please?

Reply:
I've adjusted the Ansible query. I will squash commits later into one, when everything is working :slightly_smiling_face:

Reply:
Thank you so much 🚀